stacker/config/translators/kms.py

```diff
@@ -23,9 +23,24 @@ def kms_simple_decrypt(value):
         # In stacker we would reference the encrypted value like:
         conf_key: !kms us-east-1@CiD6bC8t2Y<...encrypted blob...>
 
-        # The above would resolve to
+        You can optionally store the encrypted value in a file, ie:
+
+        kms_value.txt
+        us-east-1@CiD6bC8t2Y<...encrypted blob...>
+
+        and reference it within stacker:
+
+        conf_key: !kms file://kms_value.txt
+
+        # Both of the above would resolve to
         conf_key: PASSWORD
+
     """
+    if value.startswith('file://'):
```

Comment:
What do you think about making this more generic so that it's used by every translator?

Also is it worth using urlparse instead of the raw split you do here?  You could parse it with urlparse, and if there is no scheme, then just use the raw value?  Maybe it's not worth it at this point.